doc/deployment/azure.md

```diff
@@ -1,138 +1,384 @@
 # Azure
 
-To deploy Pachyderm to Azure, you need to:
+You can deploy Pachyderm in a new or existing Microsoft® Azure® Kubernetes
+Service environment and use Azure's resource to run your Pachyderm
+workloads. 
+To deploy Pachyderm to AKS, you need to:
 
-1. [Install Prerequisites](#prerequisites)
+1. [Prerequisites](#prerequisites)
```

Comment:
need to "prerequisites" doesn't really make sense. I think "install prereqs" as it was before made more sense.